core-requirements.txt

```diff
@@ -1,8 +1,8 @@
-numpy>=1.16.4
-pandas>=0.25.0
+numpy>=1.19.1
```

Comment:
@angela97lin what happens if you keep this PR as-is, but delete this change so that older numpy versions are still allowed? Do the other changes still solve the problem, or is updating our min numpy also required?

Reply:
@dsherry I posted in the original issue, but we get this error if we try to specifically install a version less than these because of Woodwork requirements:
```
ERROR: featuretools 0.22.0 has requirement pandas!=1.1.0,!=1.1.1,<2.0.0,>=0.24.1, but you'll have pandas 1.1.0 which is incompatible.

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
woodwork 0.0.6 requires numpy>=1.19.1, but you have numpy 1.16.4 which is incompatible.

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
woodwork 0.0.6 requires pandas>=1.1.0, but you have pandas 0.25.0 which is incompatible.
```